### PR TITLE
Bump version from `0.0.1` to `0.1.0`

### PR DESCRIPTION
**Pull request by mcdonnnj**
#  #

## 🗣 Description ##

This pull request bumps the version from `0.0.1` to `0.1.0`. It also includes any necessary changes to support using the corresponding version of [cisagov/skeleton-python-library].

## 💭 Motivation and context ##

Since this project is a Dockerization of [cisagov/skeleton-python-library] it makes sense that we should be using the latest version of that project.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] Tests have been added and/or modified to cover the changes in this PR.
- [x] All new and existing tests pass.

[cisagov/skeleton-python-library]: https://github.com/cisagov/skeleton-python-library

### CODE REVIEW

**Review by jsf9k — APPROVED** [review]

File: tests/container_test.py [hunk 1]
```diff
@@ -10,6 +10,7 @@
 
 ENV_VAR = "ECHO_MESSAGE"
 ENV_VAR_VAL = "Hello World from docker compose!"
```
Comment (dav3r) [inline tests/container_test.py:12]: Can someone tell me if/how `ENV_VAR` is used here?  I seem to recall some magic here, but I can't see it right now.  (I know this is outside of the scope of this PR, but it was gnawing at me.)

**Review by dav3r — APPROVED** [review]
👍  though still would like to get an answer to my question.

File: tests/container_test.py (hunk 1 above)
Comment (mcdonnnj) [reply tests/container_test.py:12]: It looks like that was lost in https://github.com/cisagov/skeleton-docker/commit/365fff926c7c11d266d4cc64148f2d2d9b283245 when testing was updated. Skimming over the tests I see some changes I'd like to make so I will PR to make those changes and restore the test that used that variable.

Comment (dav3r) [reply tests/container_test.py:12]: I see - thanks for finding that.

**Pull request merged**